content/tutorials/r_brms/brms_nl/workshop_1_mcmc_en_brms.Rmd

```diff
@@ -1046,10 +1049,186 @@ comp_waic %>%
+
+Meer info vind je [op het internet](https://mc-stan.org).
+
+Het voordeel van `brms` is bruikbaarheid: veel functies werken "out-of-the-box", met redelijke standaardwaarden en een syntaxis die lijkt op wat frequentisten gewend zijn.
+Het relatieve gebruiksgemak gaat echter ten koste van de flexibiliteit en in zekere mate de leesbaarheid.
+
+Stan en `rstan` daarentegen leunen meer op de wiskundige formulering van modellen.
+Elk aspect van het model moet expliciet worden ingesteld, wat een voordeel kan zijn (bijvoorbeeld als u te maken hebt met niet-standaard use cases) of een nadeel (bijvoorbeeld als u modellen op niet-optionele manieren beveiligt).
+
+Om kort een indruk te geven, bouwen we dezelfde modellen als hierboven, met behulp van het Stan-framework.
+
+
+```{r}
+library("rstan")
+conflicted::conflicts_prefer(rstan::extract)
+conflicted::conflicts_prefer(brms::loo)
+```
+
+### Model Definition
+The simple poisson model resembles [one of the `stan`dard examples](https://mc-stan.org/docs/stan-users-guide/posterior-prediction.html#posterior-prediction-for-regressions), which you can refer to for all further details and more.
+
+RMarkdown kan `stan`-chunks verwerken, hoewel de algemenere modeldefinitie is uitbesteed aan een apart "*.stan"-bestand.
+Alternatief kan je je model ook als grote tekstblok definiëren, zoals hieronder te zien.
+Het eenvoudige poissonmodel lijkt op [een van de `stan`-daardvoorbeelden](https://mc-stan.org/docs/stan-users-guide/posterior-prediction.html#posterior-prediction-for-regressions), waarnaar u terecht kunt voor verdere details en meer.
+
+
+```{r stan_poisson_simple, eval=TRUE, output.var="", class.source='fold-show'}
+stan_poisson_model_code = '
+data {
+  int<lower=1> N;                   // sample size
+  vector[N] habitat_obs;            // habitat
+  array[N] int<lower=0> sp_rich;    // outcome variable: number of ants
+}
+
+parameters {
+  real intercept_rich;              // intercept
+  real habitat_effect;              // slope
+}
+
+model {
+  // priors
+  intercept_rich ~ normal(0, 1);
+  habitat_effect ~ normal(0, 1);
+
+  // "posterior", "likelihood", ... give it a name!
+  sp_rich ~ poisson_log(intercept_rich + habitat_effect * habitat_obs);
+
+}
+
+'
+```
+
+
+
+Zie dit als een "kijkje achter de schermen"!
+Je moet expliciet de modelstructuur, priors en zelfs de gegevenstypen van invoervariabelen definiëren.
+Maar let op hoe zelfs Stan niet zonder handige functies is: we kunnen de `poisson_log` posterior gebruiken om de $log(\lambda )$ te modelleren.
+
+Als je buiten RStudio/RMarkdown werkt, kun je het model het beste laden vanuit een bestand:
+
+```{r stan_load_model, eval=TRUE, class.source='fold-show'}
+stan_poisson_model <- stan_model(
+  # file = "./poisson_model.stan",
+  model_code = stan_poisson_model_code, 
+  model_name = "stan poisson model"
+  )
+```
+
+
+### Sampling
+
+Sampling doet vrijwel hetzelfde als hierboven, want in de kern is `brms` gewoon `stan`.
+
+```{r stan_sampling}
```

Comment:
Ik zou hier ook zelf een burn-in gebruiken zoals we in de brms-voorbeelden hebben gedaan. Dan is het nog meer gelijkaardig als hierboven.

Reply:
Thank you for pointing this out, @wlangera !
I briefly asked @hansvancalster  and googled "burn-in tuning warmup bayesian stats". There is a [to-the-point discussion here](https://statmodeling.stat.columbia.edu/2017/12/15/burn-vs-warm-iterative-simulation-algorithms). My notion of changing came from the fact that we don't physically "burn" or "burn in" anything; thus I was surprised that apparently the term was considered appropriate for some MCMC algorithms. "Tuning", my actual favorite :metal:, is reserved for a certain sub-part of the warm-up phase.

Anyways, the experts seem to prefer "warm-up", and if you agree and confirm, I will in turn adjust this throughout the tutorial.

Reply:
Ok that sounds good. Thanks